local-storage-service/server/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:11
+VOLUME /tmp
```

Comment:
Please investigate what VOLUME is for (to keep data I assume?). Probably we should update this value.

Reply:
As agreed, we'll clean this up for now, directory will be added in the next task where it is actually be needed